### ISSUE TITLE
[Print Dialog] Print & cancel button getting truncated with bigger text size

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#648]
Environment Details:
Application Name: Photos, Paint

Os version and Build - Windows 11 Enterprise 24H2 (Build- 26100.4946)

pre-requisites:
Settings -> Accessibility -> Text size - 225% (max)
Repro Steps:
set Settings -> Accessibility -> Text size:  225% (max)
Open Photos
Right click any image -> Print
Or
set Settings -> Accessibility -> Text size:  225% (max)
Open folder in File Explorer
Double-click/right-click open with Photos
Ctrl+P or print button in the titlebar to print
Actual Result:
Print & cancel button getting truncated
Image
Expected Result:
Print & cancel button should be clearly visible
User Impact:
Low vision user will face difficulty if Print & cancel button getting truncated at resize resolution.
MAS Reference:
[MAS 1.4.4 – Resize Text](https://aka.ms/MAS1.4.4)

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.
